persist-cli-tests/src/test/java/io/ballerina/persist/tools/ToolingGenerateTest.java

```diff
@@ -68,10 +68,10 @@ public void testGenerateAllEntityFieldTypes() {
         assertGeneratedSources("tool_test_generate_6", GENERATE);
     }
 
-    @Test(enabled = true) // this testcase not valid.
-    @Description("Use case where a entity is located inside a module")
-    public void testGenerateClientWithEntityInModule() {
-        assertGeneratedSources("tool_test_generate_7", GENERATE);
+    @Test(enabled = true) //disabled sue to compiler plugin changes
```

Comment:
```suggestion
    @Test(enabled = true)
```

Reply:
Do we need this commend now

Reply:
No we don't it was added before updating to the latest compiler plugin